assets/js/modules/reader-revenue-manager/components/common/PublicationCreate.js

```diff
@@ -84,9 +89,23 @@ export default function PublicationCreate( { onCompleteSetup } ) {
 						) }
 					</h3>
 					<p className="googlesitekit-setup-module__description">
-						{ __(
-							'Once you have created your publication, it is submitted for review.',
-							'google-site-kit'
+						{ createInterpolateElement(
+							__(
+								'Once you have created your publication, it is submitted for review. <a>Learn more</a>',
+								'google-site-kit'
+							),
+							{
+								a: (
+									<SupportLink
+										path="/news/publisher-center/answer/11449914"
+										external
+										aria-label={ __(
+											'Learn more about publisher center',
```

Comment:
Let's make this label a bit more focused:

```suggestion
											'Learn more about setting up Reader Revenue Manager',
```